src/conv-core/threads.C

```diff
@@ -456,7 +457,9 @@ static void *CthAllocateStack(CthThreadBase *th, int *stackSize, int useMigratab
   if (*stackSize==0) *stackSize=CthCpvAccess(_defaultStackSize);
   th->stacksize=*stackSize;
   if (!useMigratable || !CmiIsomallocEnabled()) {
-    ret=malloc(*stackSize); 
+    CmiEnforce(*stackSize <= std::numeric_limits<size_t>::max());
```

Comment:
Unfortunately this line is a tautological comparison. A `size_t` will always be <= the max value possible for the type. LGTM did not consider this to fix the alert. I suggest we use the max value of `uint32_t` here instead of `size_t`, since a 4 GB stack is absurdly large.

Reply:
Actually, I see now that `stackSize` is an `int`. If you change `size_t` to `int` the warning should go away.

Reply:
```suggestion
    CmiEnforce(*stackSize <= std::numeric_limits<int>::max());
```